src/View/SSViewer.php

```diff
@@ -296,7 +296,7 @@ public static function get_themes()
     }
 
     /**
-     * @deprecated 4.0.0:5.0.0 Use the "SSViewer#set_themes" instead
+     * @deprecated 4.0.0 Use the "SSViewer#set_themes" instead
```

Comment:
```suggestion
     * @deprecated 4.0.0 Use SSViewer::set_themes() instead
```
Also update the notice message below

Reply:
Updated